src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -424,9 +427,119 @@ export class StudyViewPageStore {
         this._comparisonGroupsChangeCount += 1;
     }
 
-    private createPieChartComparisonSession(
+    private async createNumberAttributeComparisonSession(
         clinicalAttribute:ClinicalAttribute,
-        clinicalAttributeValues:{ value:string, color:string}[],
+        statusCallback:(phase:LoadingPhase)=>void
```

Comment:
probably really late to the game here, but is there no way we could accomplish this with our usual status tracking mechanism (networks for mobx promises).  just wanted to discuss.